buildspec-e2e-pr-join-artifacts.yml

```diff
@@ -0,0 +1,15 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - ls -la $CODEBUILD_SRC_DIR # rc branch, also called as primary source
+      - ls -la $CODEBUILD_SRC_DIR_SourceArtifact # topic branch
```

Comment:
Do we still need these directory loggers? Do you think we will need them in the future?

Reply:
Not sure if we need them anymore, there were there before. I will remove them